### DISCUSSION TITLE
Any Replacement for A60/A70 speakers?

### DISCUSSION THREAD

**Discussion by oddomatik** [shorepine/alles#73] [Q&A]
These are broken links to alibaba, are they still available under a different name? Thanks.

**Comment by oddomatik** [comment]
I think I found some.
https://www.alibaba.com/product-detail/popular-subwoofer-wireless-speaker-wooden-style_1600321487036.html?spm=a2700.galleryofferlist.normal_offer.d_title.13e8581aYperug

**Comment by bwhitman** [comment]
Thanks for finding them! They're getting cheaper every day! If you end up ordering them and have success, let me know and i'll update the docs.
